project/plugins.sbt

```diff
@@ -1,3 +1,2 @@
-addSbtPlugin("com.dwijnand"      % "sbt-dynver"      % "4.1.1")
```

Comment:
We want to keep this, though - it's a really handy plugin! 😄

Reply:
I know :wink: However, it's enough to just add `sbt-ci-release`, since that pulls in sbt-dynver and other dependencies for you:
> By installing sbt-ci-release the following sbt plugins are also brought in:
> * [sbt-dynver](https://github.com/dwijnand/sbt-dynver): sets the version number based on your git history
...

https://github.com/sbt/sbt-ci-release#sbt

So there is no need to explicitly add sbt-dynver (I did lot's of sbt-ci-release migrations and know that works)

Reply:
True.  I thought I used this to test the release, but I realise I do that in the the release of MiMa.